### ISSUE TITLE
Publish new release?

### ISSUE THREAD

**Issue by repi** [amodm/webbrowser-rs#23]
Possible to publish a new release of the crate?

We are specifically after the Android support that came in #19

**Comment by amodm** [comment]
It's currently pending on testing on Android, which I haven't gotten around to do. Would you be able to help me with that? To do that, you can include the Android support (temporarily) by modifying your Cargo.toml

```rust
[dependencies]
webbrowser = { git="https://github.com/amodm/webbrowser-rs", rev="75fad4d" }
```

**Comment by repi** [comment]
I think @VZout in our team has been testing this on Android, so he should be able to confirm if it works

**Comment by amodm** [comment]
@VZout, any visibility into this?

**Comment by VZout** [comment]
Sorry for the late reply. Must have missed the notification.
It compiles and links but I haven't tested it. I'll give you an update tomorrow!

**Comment by VZout** [comment]
@amodm Works like a charm!

**Comment by amodm** [comment]
I've released 0.5.4 for this. Thanks @VZout for the help on testing.
